st2actions/st2actions/query/mistral/v2.py

```diff
@@ -129,8 +129,8 @@ def _format_task_result(self, task):
 
         result['id'] = task['id']
         result['name'] = task['name']
-        result['execution_id'] = task.get('execution_id', None)
-        result['workflow'] = task['wf_name']
+        result['workflow_execution_id'] = task.get('workflow_execution_id', None)
```

Comment:
@Kami FYI, I'm going to have to push this to 0.8.1 as well.  There's a bug fix we need in mistral.  But this backward compatibility change superseded the bug fix in mistral.